scripts/testing/tolerance_test.py

```diff
@@ -37,32 +37,74 @@ def ensure_file(path):
         print("ERROR: Given file is not a file: {0}".format(path))
         sys.exit(1)
 
-
 def parse_args():
-    parser = argparse.ArgumentParser(description="Compare two Serac summary files")
+    usage = """
+Compare two Serac summary files.
+
+Example usages for --tolerance:
+    --tolerance=0.001 // used for all field values
+    --tolerance=default:0.001,velocity:0.1,displacement:0.234 // default is for all non-specified tolerances
+
+Example usage for --tolerance-file:
+    --tolerance-file=tolerance_low.json
+    (Note: <name>.json file must be in tests/integration/tolerances directory)
```

Comment:
related to comment in other PR: 

https://github.com/LLNL/serac_tests/pull/4/files#r854621606